coala-bears.cabal

```diff
@@ -10,4 +10,4 @@ build-type: Simple
 -- which is incompatible with ShellCheck 0.4.1 which needs mtl-2.2.1
 
 library
-  build-depends: base, Cabal>=1.16, safe==0.3.9, cmdargs==0.10.14, ghc-mod==5.6.0.0, hlint==1.9.27, ShellCheck==0.4.1
+  build-depends: base, Cabal>=1.16, safe==0.3.9, cmdargs==0.10.14, ShellCheck==0.4.1, ghc-mod==5.6.0, hlint==1.9.27
```

Comment:
oh its failing.
probably because ShellCheck needs to be last.

Manually reorder them in `bear-requirements.yaml` , and leave a note that the order is needed for cabal to work

Reply:
i tested this.. it still didn't work
the version for `ghc-mod` should be `5.6.0.0` rather than `5.6.0` as specified